### PR TITLE
Update config-mattermost-server.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Updates to formatting, numbering, removed reference to 5.12/prior version, updated restart command to match upgrade guide.

Note: I removed reference to "5.12/previous versions" as this is an installation guide so I assumed users would be using the latest version.

### CODE REVIEW

**Review by svelle — APPROVED** [review]
LGTM

File: source/install/config-mattermost-server.rst [hunk 1]
```diff
@@ -5,53 +5,48 @@ Create the System Admin user and set up Mattermost for general use.
...
+  * In the **Site URL** field, set the URL that users point their browsers at. For example, *https://mattermost.example.com*. If you are using HTTPS, make sure that you set up TLS, either on Mattermost Server or on a proxy.
 
 5. Set up email notifications.
 
-  a. In the NOTIFICATIONS section of the System Console, make the following changes:
+  * In **Site Configuration > Notifications** make the following changes:
 
-    - Set **Enable Email Notifications** to *true*
-    - Set **Notification Display Name** to *No-Reply*
+    - Set **Enable Email Notifications** to **true***
```
Comment (amyblais) [inline source/install/config-mattermost-server.rst:21]:
```suggestion
    - Set **Enable Email Notifications** to **true**
```

**Review by amyblais — APPROVED** [review]

File: source/install/config-mattermost-server.rst [hunk 2]
```diff
@@ -5,53 +5,48 @@ Create the System Admin user and set up Mattermost for general use.
 
 1. Open a browser and navigate to your Mattermost instance. For example, if the IP address of the Mattermost server is ``10.10.10.2`` then go to http://10.10.10.2:8065.
 
-2. Create the first team and user. The first user in the system has the ``system_admin`` role, which gives you access to the System Console.
+2. Create the first team and user. The first user in the system has the System Admin role, which gives you access to the System Console.
```
Comment (justinegeffen) [inline source/install/config-mattermost-server.rst:8]:
```suggestion
2. Create the first team and user. The first user in the system has the ``system_admin`` role, which gives you access to the System Console.
```

**Comment by justinegeffen** [comment]
@thefactremains, I am going to merge this as I need the updated content for a guide that I'm putting together. These are mainly structural changes, for ease of reading. LMK if you have any additional comments and I'll open a new PR. :)

**Pull request merged**